### PR TITLE
Implement dummy OTLP logs and trace services

### PR DESCRIPTION
**Pull request by guilload**
### Description
- Setup OpenTelemetry OTLP exporter
- Implement dummy OTLP logs and trace services

### How was this PR tested?
1. Enabled OTLP services in indexer config
2. Ran the following command:
```bash
QW_ENABLE_OPENTELEMETRY_OTLP_EXPORTER=1 OTEL_EXPORTER_OTLP_ENDPOINT=http://127.0.0.1:7281 cargo  run --manifest-path quickwit/Cargo.toml -- run
```

### CODE REVIEW

**Comment by guilload** [comment]
PTAL!

File: quickwit/quickwit-actors/benches/bench.rs [hunk 1]
```diff
@@ -75,7 +75,7 @@ async fn flume_bench_code(num_messages: usize) {
     }
     let join = tokio::task::spawn(async move {
         let mut sum = 0;
-        while let Ok(_) = rx.recv_async().await {
+        while (rx.recv_async().await).is_ok() {
```
Comment (fmassot) [inline quickwit/quickwit-actors/benches/bench.rs:78]: I think it's already committed

File: quickwit/quickwit-cli/src/main.rs [hunk 2]
```diff
@@ -62,23 +66,43 @@ fn setup_logging_and_tracing(level: Level) -> anyhow::Result<()> {
...
-        // TODO: use install_batch once this issue is fixed: https://github.com/open-telemetry/opentelemetry-rust/issues/545
+    if std::env::var_os(QW_ENABLE_JAEGER_EXPORTER_ENV_KEY).is_some() {
         let tracer = opentelemetry_jaeger::new_agent_pipeline()
             .with_service_name("quickwit")
-            //.install_batch(opentelemetry::runtime::Tokio)
-            .install_simple()
+            .install_batch(opentelemetry::runtime::Tokio)
             .context("Failed to initialize Jaeger exporter.")?;
         registry
+            .with(tracing_opentelemetry::layer().with_tracer(tracer))
```
Comment (fmassot) [inline quickwit/quickwit-cli/src/main.rs:75]: why do we need this additional layer?

Comment (fmassot) [reply quickwit/quickwit-cli/src/main.rs:75]: oh it's just the diff, it was there before, my bad

**Review by fmassot — APPROVED** [review]

**Pull request merged**